routes.txt

```diff
@@ -1224,7 +1224,6 @@
 /platforms/stellar-disbursement-platform/admin-guide/making-your-wallet-sdp-ready
 /platforms/stellar-disbursement-platform/admin-guide/overview
 /platforms/stellar-disbursement-platform/admin-guide/secure-operation-manual
-/platforms/stellar-disbursement-platform/admin-guide/single-tenant-to-multi-tenant-migration
```

Comment:
Is there a page that we should redirect any requests to? Adding one can help anybody who's bookmarked this page, and helps with SEO if the page has been indexed (i think? i'm not much of a marketing guy lol)

A redirect can be added to `/nginx/includes/redirects.conf`. If you need help taking care of that, I'd be more than happy to lend a hand!